### ISSUE TITLE
Consider Tree flow chart for decision making

### ISSUE THREAD

**Issue by AARON-CLARK** [pharmaR/riskassessment#601]
Per feedback from useR@ 2023 conf in Basel, @Xyarz reported the desire for this issue. Did I describe it correctly Lars?

I can confirm, there are all sorts of pharma orgs using these tree diagrams to make their GxP inclusion decisions using automated quality checks, especially in the [2022 case studies](https://www.pharmar.org/casestudies/). Nodes would have to absolutely be set up using the config file. I've pasted some examples below!

![image](https://github.com/pharmaR/riskassessment/assets/17878953/86b1075b-9b3d-45b9-8d79-796fac89134d)

![image](https://github.com/pharmaR/riskassessment/assets/17878953/17a11717-e81d-4b39-b617-59c9199c2a0c)

![image](https://github.com/pharmaR/riskassessment/assets/17878953/a8233678-a684-495c-adc3-a914147052e9)

**Comment by Xyarz** [comment]
Yes! Thanks @AARON-CLARK. The ask was regarding a tree based algorithmic decision.

**Comment by AARON-CLARK** [comment]
> ## User feedback
> So at the end of the day, how do we help users understand how the decision automation rule's code was evaluated? Right now, it's very "black boxie".
> 
> 1. If a decision isn't made, that means all of the rules ran and nothing returned `TRUE`. I'm not sure what we should display to reflect this. Doing nothing is okay for now, but what if I expected something to happen (like in the example above) but don't have a way to verify the results? Do you have any thoughts? In an ideal world, It'd be nice to show both the `.x` value and the resulting `TRUE`/`FALSE` for each condition (rule), but that's difficult to display in the Upload Summary Table. Perhaps we could tackle it in a future PR.

_Originally posted by @AARON-CLARK in https://github.com/pharmaR/riskassessment/pull/632#pullrequestreview-1646627748_

> ## User feedback
> This is a big question that I am still struggling with. I suppose we should at least store dummy assessment data and verify that the rules will actually run before allowing a user to add them to the database. I will think more on how to display what occurred when a package is uploaded.

_Originally posted by @Jeff-Thompson12 in https://github.com/pharmaR/riskassessment/issues/632#issuecomment-1737621898_
